lib/puppet_x/sqlserver/sql_connection.rb

```diff
@@ -90,24 +93,25 @@ def win32_exception
     end
 
     class ResultOutput
-      attr_reader :exitstatus, :error_message, :raw_error_message
 
-      def initialize(has_errors, error_message)
+      attr_reader :exitstatus, :error_message
+
+      def initialize(has_errors, error_message, connection)
```

Comment:
Warning, this variable has the same name as a method

Reply:
How does ole_connection sound?

Reply:
@RandomNoun7 its the `@has_errors` bit that should be changed (which predates your code)... `connection` is fine

Reply:
Ahh. I can change it now, or we can call it out of scope for the ticket. I'm fine either way.

Reply:
Derp.....I should've read more carefully.  I was meaning the `has_errors` but you didn't change that.

Need more ☕️

Reply:
I can change the name of the method to ```errors?``` and it doesn't seem to break anything.

Reply:
Will rename or remove in a maint commit.